include/wx/osx/core/private.h

```diff
@@ -741,7 +741,7 @@ public :
     virtual wxString GetLineText(long lineNo) const ;
 #if wxUSE_SPELLCHECK
     virtual void CheckSpelling(const wxTextProofOptions& WXUNUSED(options)) { }
-    virtual bool IsSpellingCheckEnabled() const { return false; }
+    virtual wxTextProofOptions GetCheckingOptions() const;
```

Comment:
Can't this be GetProofCheckOptions() too or is it for backwards compatibility with an existing function?